tandem/logger.py

```diff
@@ -11,7 +11,7 @@
 from typing import Optional
 
 from arc.common import time_lapse
-
```

Comment:
there should be an empty line between imports related to external packages (ARC) and internal imports